tests/code_wars/test_luck_check.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+from code_challenges.code_wars.luck_check import luck_check
+
+
+def test_luck_check_true():
+    actual = luck_check("683179")
+    expected = True
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_luck_check_false():
+    actual = luck_check("683000")
+    expected = False
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_luck_check_odd_length():
+    actual = luck_check("6838179")
+    expected = True
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_luck_check_empty_string():
+    with pytest.raises(ValueError) as exc_info:
+        luck_check("")
+        assert exc_info.value.args[0] == "empty string"
```

Comment:
The `assert` needs to be pulled outside the `with` block.  To see what I mean you can change `== "empty string"` too `== "foo"`, it should fail but won't.

As soon as luck_check is ran the value error is thrown. Pytest catches this error and exits the with context block.